db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_01_21_130904) do
+ActiveRecord::Schema.define(version: 2020_06_11_102258) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
```

Comment:
To enable uuid generation on the DB you would need to add the postgres extension
```suggestion
  enable_extension "plpgsql"
  enable_extension "uuid-ossp"
```